Makefile

```diff
@@ -0,0 +1,29 @@
+include ./env_make
```

Comment:
what's the benefit of having those variables in another file?

Reply:
makefile is totally portable and can be dropped as is into any project. 

If you use it with an s2i based build you need one of those files with the env_vars in every context folder and then you can use a single makefile to build all versions of the project (admittedly the s2i version has a small alteration to this version in that it looks for the env vars in the context folder that you pass to the make command, but that is literally the only difference)